src/components/primitive/accordion/accordion.component.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import { Box, IconSvg, Text } from 'components/primitive';
+import { AccordionProps } from './accordion.component.props';
+import { AccordionContainer, IconContainer, LabelContainer } from './accordion.component.styles';
+
+export const Accordion: React.FC<AccordionProps> = (props) => {
+  const { label, children } = props;
+  const [isActive, setIsActive] = useState(false);
+
+  return (
+    <AccordionContainer>
+      <LabelContainer>
+        <Text>{label}</Text>
+        <IconContainer align={'end'} onClick={() => setIsActive(!isActive)}>
+          <IconSvg name={isActive ? 'minus' : 'add'} size='xLarge' />
```

Comment:
Let's use a different icon here. I have updated in design:
![updated option](https://user-images.githubusercontent.com/4023530/160078504-0c978e97-8c87-4bc5-85df-909110cc4d08.png)

Reply:
okay, will update it👍